test-site/__tests__/integration/schema/data-resolution.test.js

```diff
@@ -52,96 +51,96 @@ describe(`[gatsby-source-wordpress-experimental] data resolution`, () => {
     },
   })
 
-  testResolvedData({
-    url,
-    title: `resolves wp-graphql-gutenberg columns`,
-    gatsbyQuery: queries.gutenbergColumns,
-    queryReplace: {
-      from: `wpPost(title: { eq: "Gutenberg: Columns" }) {`,
-      to: `post(id: "cG9zdDoxMjg=") {`,
-    },
-    fields: {
-      gatsby: `wpPost`,
-      wpgql: `post`,
-    },
-  })
-
-  testResolvedData({
-    url,
-    title: `resolves wp-graphql-gutenberg layout elements`,
-    gatsbyQuery: queries.gutenbergLayoutElements,
-    queryReplace: {
-      from: `wpPost(id: { eq: "cG9zdDoxMjU=" }) {`,
-      to: `post(id: "cG9zdDoxMjU=") {`,
-    },
-    fields: {
-      gatsby: `wpPost`,
-      wpgql: `post`,
-    },
-  })
-
-  testResolvedData({
-    url,
-    title: `resolves wp-graphql-gutenberg formatting blocks`,
-    gatsbyQuery: queries.gutenbergFormattingBlocks,
-    queryReplace: {
-      from: `wpPost(id: { eq: "cG9zdDoxMjI=" }) {`,
-      to: `post(id: "cG9zdDoxMjI=") {`,
-    },
-    fields: {
-      gatsby: `wpPost`,
-      wpgql: `post`,
-    },
-  })
-
-  testResolvedData({
-    url,
-    title: `resolves wp-graphql-gutenberg common blocks`,
-    gatsbyQuery: queries.gutenbergCommonBlocks,
-    queryReplace: {
-      from: `wpPost(id: { eq: "cG9zdDo5NA==" }) {`,
-      to: `post(id: "cG9zdDo5NA==") {`,
-    },
-    fields: {
-      gatsby: `wpPost`,
-      wpgql: `post`,
-    },
-  })
-
-  it(`resolves Yoast SEO data`, async () => {
-    const gatsbyResult = await fetchGraphql({
-      url,
-      query: /* GraphQL */ `
-        {
-          wp {
-            ${queries.yoastRootFields}
-          }
-          wpPage(title: {eq: "Yoast SEO"}) {
-            ${queries.pageYoastFields}
-          }
-        }
-      `,
-    })
-
-    const WPGraphQLData = await authedWPGQLRequest(/* GraphQL */ `
-      {
-        ${queries.yoastRootFields}
-        page(id: "cG9zdDo3ODY4") {
-          ${queries.pageYoastFields}
-        }
-      }
-    `)
-
-    const wpGraphQLPageNormalizedPaths = JSON.parse(
-      JSON.stringify(WPGraphQLData.page).replace(
-        /https:\/\/gatsbyinttests.wpengine.com/gm,
-        ``
-      )
-    )
-
-    expect(gatsbyResult.data.wpPage).toStrictEqual(wpGraphQLPageNormalizedPaths)
-    expect(gatsbyResult.data.wp.seo).toStrictEqual(WPGraphQLData.seo)
-  })
+  // testResolvedData({
```

Comment:
@TylerBarnes delete or todos? i guess we can address that in another PR?

Reply:
I was going to comment the same. Delete commented out bits to keep it clean.